packages/shylu/shylu_dd/frosch/test/CMakeLists.txt

```diff
@@ -7,5 +7,9 @@ ADD_SUBDIRECTORIES(
     LocalPartitionOfUnityBasis
 #    InterfacePartitionOfUnity
     TwoLevelPreconditionerInputFiles
-    TwoLevelPreconditionerBlockInputFiles    
+    Thyra_Xpetra
+    Thyra_Xpetra_Prec
+    Thyra_Tpetra
+#  Thyra_Xpetra_PrecBlock
+#  TwoLevelPreconditionerBlockInputFiles
```

Comment:
@chochmuth  What is the reason that the block tests are disabled?

Reply:
@searhein I disabled them because there were no `#ifdef`-checks for EpetraExt and HDF5. They can be enabled again, since `#ifdef`-checks are in PR #3246

Reply:
This means that we can enable both tests now?

Reply:
After a discussion with @chochmuth, we decided to postpone this to the next PR.